src/app/parcels/ActionBar/StatusesModal.tsx

```diff
@@ -73,18 +67,7 @@ const StatusesBarModal: React.FC<StatusesBarModalProps> = (props) => {
                     <TimePicker value={date} onChange={onTimeChange} disableFuture />
                 </Row>
                 Applying To:
```

Comment:
We have "_ Parcels selected" as part of ShowParcels. Do we still want "Applying to:"?

Reply:
Good spot!!